{
  "contest": 207,
  "handle": "tsvetanovv",
  "risk": "3",
  "title": "Lack of access control in `Lendgine.sol.withdraw()`",
  "issueId": 162,
  "issueUrl": "https://github.com/code-423n4/2023-01-numoen-findings/issues/162"
}